src/Main.java

```diff
@@ -2840,7 +3092,15 @@ else if (region == 0x4) {
                     return inventory.contains(Items.VALVE) && canLift(inventory);
                 }
                 else {
-                    return difficulty > Difficulty.EASY && inventory.contains(Items.VALVE) && canLift(inventory) && inventory.contains(Items.JUMP_BOOTS);
+				/* 
+				* Added MINOR GLITCHES logic for the Vampire Area - Center Left check.
+				* This check can be obtained without a Glove, using just Boots.
+				* Perform a High Walljump to reach the Spearhead located in the upper right corner.
+				* Stun it, then walk into it towards the left and knock it down the ledges necessary.
+				* Once it's off the ledge that the door leading to the Hammerbot room is on, do the jumps across like normal to reach the check.
+				*/
+                    return (difficulty > Difficulty.EASY && inventory.contains(Items.VALVE) && canLift(inventory) && inventory.contains(Items.JUMP_BOOTS)
```

Comment:
This is missing a `)` somewhere, but I'd restructure this anyway - better to just have it the way it was before, but replace `canLift(inventory)` with `(difficulty >= Difficulty.S_HARD || canLift(inventory))`

Reply:
This will be restructured to
`
return difficulty > Difficulty.EASY 
	&& inventory.contains(Items.VALVE) 
	&& inventory.contains(Items.JUMP_BOOTS)
        && (difficulty >= Difficulty.S_HARD || canLift(inventory));
`